Add License (cc4.0) to repo root

rogierslag opened the pull request:
Closes https://github.com/delftswa/delftswa.github.io/issues/16

@avandeursen Ready for review!

rogierslag committed:
Add License (cc4.0) to repo root

RickWieman commented:
I actually meant a license on the website itself, not just a license in this repo

rogierslag commented:
This will also be copied to the website, it will just need a link

RickWieman commented:
Check. Let's not forget about that then :blush:

rogierslag commented:
That will happen in the intro text which will be written by @avandeursen. I will put it in the issue as well ;)

RickWieman commented:
:+1:

RickWieman commented on LICENSE.md:
```diff
@@ -0,0 +1,161 @@
+## creative commons
+
+# Attribution 4.0 International
```
These headings are rather strange; first H2, then H1 and later on H3...?

avandeursen commented:
I have this covered in the introduction that I'm writing -- issue #14. Almost done.

So let's wait for a #14, and then see if that adequately addresses the license.

RickWieman commented:
I like the way Arie wrote about the license in #19, and I don't think the introduction requires a link to this document (Arie references the license itself directly, which I think suffices).
I do think this `LICENSE.md` should be in the repo root, as that's more clearer from a repository point of view (i.e. not all GitHub viewers will read/see Arie's introduction).
That's basically what this PR does right now, though.

rogierslag commented:
I think both PR's are a great addition to the book in the end form. On the website, the portion of @avandeursen should be sufficient, whereas on Github itself the `LICENSE.md` might be clearer.

RickWieman commented:
That's what I said, right? :smile:

rogierslag commented:
Yes but shorter. Arie is a very busy man ;)

avandeursen committed:
Cleaned up top most header levels.

To bring it in sync with intent of
http://creativecommons.org/licenses/by/4.0/legalcode

avandeursen commented:
Thanks for good discussion and insight. Adjusted the opening header as suggested by @RickWieman.

RickWieman commented:
:+1:

rogierslag commented:
:+1:
